src/react.tsx

```diff
@@ -112,7 +112,7 @@ type GetInspectorModeProps<T> = (props: {
  * Generates the function to build the required properties for the inspector mode (field tagging)
  */
 export function useContentfulInspectorMode<
-  T = undefined | Pick<LivePreviewProps, 'entryId'> | Pick<LivePreviewProps, 'entryId' | 'locale'>
+  T = undefined | Pick<LivePreviewProps, 'entryId'> | Pick<LivePreviewProps, 'entryId' | 'fieldId'>
```

Comment:
Probably we could also remove the last block `Pick<LivePreviewProps, 'entryId' | 'fieldId'>` because the function will not be re-useable anymore? or we would need to adjust the typing for the `GetInspectorModeProps` because currently you would still need to provide an object as param 🤔